x11/lightdm-kde-greeter/files/patch-pragma-it

```diff
@@ -0,0 +1,11 @@
+--- greeter/keyboard/XcbKeyboardBackend.h.orig
++++ greeter/keyboard/XcbKeyboardBackend.h
+@@ -4,6 +4,8 @@
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ ***************************************************************************/
+ 
++#pragma clang diagnostic ignored "-Wkeyword-macro"
++
+ #ifndef XCBKEYBOARDBACKEND_H
+ #define XCBKEYBOARDBACKEND_H
+ 
```

Comment:
To make this patch canonically named do `make clean patch && rm files/patch-pragma-it && make makepatch`.